scripts/build_scenario1.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
All of these files can be actually a single file, where you define a function:
build_scenario(...)
which takes as input the configuration of the scenarios.

Then you define one variable per scenario:
SCENARIO_1 = {}

and you can then run your script iteratively